packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -1435,7 +1436,7 @@ describe("renderer", () => {
             expect(input).toStrictEqual([
                 {currentValue: "0"},
                 {currentValue: "1"},
-                null, // image widget doesn't implement getUserinput
+                undefined, // image widget doesn't implement getUserinput
```

Comment:
This would require us to keep external-facing methods on React components that don't need those specific external-facing methods.

I know you mentioned [in another comment](https://github.com/Khan/perseus/pull/1715#discussion_r1790721458) that you think we should keep functionality even when it's a no-op. I can kind of see your point (even if I don't agree), but I definitely don't think we want to do that with React anti-patterns (attaching external-facing methods to React components).

I suppose we could do `.getUserInput() ?? null`. Maybe that would be better than leaving it as `undefined`? It would be the tiniest bit more complexity, but at least it'd be compatible with existing behavior.